raiden/ui/cli.py

```diff
@@ -765,120 +754,94 @@ def prompt_account(address_hex, keystore_path, password_file):
     return address_hex, privatekey_bin
 
 
-@group(invoke_without_command=True, context_settings={'max_content_width': 120})
-@options
-@click.pass_context
-def run(ctx, **kwargs):
-    # pylint: disable=too-many-locals,too-many-branches,too-many-statements
+class NodeRunner:
+    def __init__(self, options: Dict[str, Any], ctx):
+        self._options = options
+        self._ctx = ctx
+        self._raiden_api = None
+
+    @property
+    def _welcome_string(self):
+        return f"Welcome to Raiden, version {get_system_spec()['raiden']}!"
+
+    def _startup_hook(self):
+        """ Hook that is called after startup is finished. Intended for subclass usage. """
+        pass
+
+    def _shutdown_hook(self):
+        """ Hook that is called just before shutdown. Intended for subclass usage. """
+        pass
+
+    def run(self):
+        click.secho(self._welcome_string, fg='green')
+        click.secho(
+            textwrap.dedent(
+                '''\
+                ----------------------------------------------------------------------
+                | This is an Alpha version of experimental open source software      |
+                | released under the MIT license and may contain errors and/or bugs. |
+                | Use of the software is at your own risk and discretion. No         |
+                | guarantee whatsoever is made regarding its suitability for your    |
+                | intended purposes and its compliance with applicable law and       |
+                | regulations. It is up to the user to determine the software´s      |
+                | quality and suitability and whether its use is compliant with its  |
+                | respective regulatory regime, especially in the case that you are  |
+                | operating in a commercial context.                                 |
+                ----------------------------------------------------------------------''',
+            ),
+            fg='yellow',
+        )
+        configure_logging(
+            self._options['log_config'],
+            log_json=self._options['log_json'],
+            log_file=self._options['log_file'],
+        )
 
-    if ctx.invoked_subcommand is not None:
-        # Pass parsed args on to subcommands.
-        ctx.obj = kwargs
-        return
+        # Do this here so logging is configured
```

Comment:
I don't understand this comment, will the logging not be configured if this was called somewhere else?

Reply:
It used to make more sense before the refactor, it simply explains why the log message is done there and not at the time when the config file is actually being processed.
I've removed it.